tasks/system_probe.py

```diff
@@ -53,13 +53,14 @@
     "./pkg/network/...",
     "./pkg/collector/corechecks/ebpf/...",
     "./pkg/process/monitor/...",
-    "./pkg/usm/...",
 ]
 TEST_PACKAGES = " ".join(TEST_PACKAGES_LIST)
+# change `timeouts` in `test/new-e2e/system-probe/test-runner/main.go` if you change them here
 TEST_TIMEOUTS = {
-    "pkg/network/usm$": "0",
-    "pkg/network/protocols/http$": "0",
     "pkg/network/protocols": "5m",
+    "pkg/network/protocols/http$": "15m",
+    "pkg/network/tracer$": "55m",
+    "pkg/network/usm$": "55m",
```

Comment:
we should have a specific entry to the new sub-package
today USM tests (prior to the change) takes around 30m, and the classification tests are heavy as well, I have a concern that we will see more "test timeouts" after this change

Reply:
The data shows a max of 7.8 min for `TestUSMSuite`, so I think the default of 10 minutes is OK for now. WDYT?

Reply:
Let's make it 20m, to ensure we'll not break the limit for every test
also, many TLS classification tests are disabled now, and we plan to re-enable them after fixing the flakiness

Reply:
Good point about disabled tests. Increased.